src/frontend/Komponenter/Behandling/Brev/ManueltBrev.tsx

```diff
@@ -0,0 +1,177 @@
+
+        const brevdato = dagensDatoFormatert();
+
+        axiosRequest<string, IManueltBrev>({
+            method: 'POST',
+            url: `/familie-ef-sak/api/manueltbrev`,
+            data: {
+                overskrift,
+                avsnitt,
+                saksbehandlersignatur: innloggetSaksbehandler?.displayName || 'Ukjent',
+                brevdato,
+                ident: personopplysninger.data.personIdent,
+                navn: personopplysninger.data.navn.visningsnavn,
+            },
+        }).then((respons: Ressurs<string>) => {
+            settBrev(respons);
+        });
+    };
+
+    const leggTilRad = () => {
+        settAvsnitt((eksisterendeAvsnitt: IAvsnitt[]) => {
+            return [
+                ...eksisterendeAvsnitt,
+                {
+                    deloverskrift: '',
+                    innhold: '',
+                },
+            ];
+        });
+    };
+
+    const endreDeloverskrift = (e: React.ChangeEvent<HTMLInputElement>) => {
+        e.preventDefault();
+
+        const index = parseInt(e.target.id, 10);
+
+        settAvsnitt((eksisterendeAvsnitt: IAvsnitt[]) => {
+            const oppdatertAvsnitt = eksisterendeAvsnitt.slice();
+
+            oppdatertAvsnitt[index].deloverskrift = e.target.value;
+
+            return oppdatertAvsnitt;
+        });
+    };
+
+    const endreInnhold = (e: React.ChangeEvent<HTMLInputElement>) => {
+        e.preventDefault();
+
+        const index = parseInt(e.target.id, 10);
+
+        settAvsnitt((eksisterendeAvsnitt: IAvsnitt[]) => {
+            const oppdatertAvsnitt = eksisterendeAvsnitt.slice();
+
+            oppdatertAvsnitt[index].innhold = e.target.value;
+
+            return oppdatertAvsnitt;
+        });
+    };
+
+    return (
+        <StyledManueltBrev>
+            <h1>Manuelt brev</h1>
+            <BrevKolonner>
+                <div>
+                    <Input
+                        label="Overskrift"
+                        value={overskrift}
+                        onChange={(e) => {
+                            settOverskrift(e.target.value);
+                        }}
+                    />
+
+                    {avsnitt.map((rad, i) => {
+                        return (
+                            <Innholdsrad border>
+                                <Input
+                                    onChange={endreDeloverskrift}
+                                    label="Deloverskrift (valgfri)"
+                                    id={i.toString()}
+                                    value={rad.deloverskrift}
+                                />
+                                <Textarea
+                                    onChange={endreInnhold}
+                                    defaultValue=""
+                                    label="Innhold"
+                                    id={i.toString()}
+                                    value={rad.innhold}
+                                />
```

Comment:
Kanskje legge på dette så vi slipper "2000 tegn igjen"
```suggestion
                                <Textarea
                                    onChange={endreInnhold}
                                    defaultValue=""
                                    label="Innhold"
                                    id={i.toString()}
                                    value={rad.innhold}
                                    maxLength={0}
                                />
```

Reply:
Jepp, fikset i 48223c1